src/modules/admin/__tests__/mentors.controller.spec.ts

```diff
@@ -1,5 +1,6 @@
 import { UnauthorizedException, BadRequestException } from '@nestjs/common';
 import { Test } from '@nestjs/testing';
+import { Request } from '🧙‍♂️/types/request';
```

Comment:
lol, what's with the 🧙‍♂️ in the filepath?

Reply:
I was exciting to use tsconfig/paths for the first time :)

https://github.com/Coding-Coach/find-a-mentor-api/blob/bc67dedc4a6095d926094b3c78a84309aa6ade2a/tsconfig.json#L14-L16

🧙‍♂️ is our `paths` root